code/src/iamf_dec/m2m_rdr.c

```diff
@@ -25,6 +25,11 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+/*
+AOM-IAMF Standard Deliverable Status:
+The following function is out of scope and not part of the IAMF Final Deliverable.
```

Comment:
Please replace this line by:
This software module is out of scope and not part of the IAMF Final Deliverable.